### PR TITLE
[Notification Center] Refactor EVENT_TYPES and Icons

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**
- Removed duplication of eventtypes and there icons  

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by aabidsofi19** [comment]
@theBeginner86  this also fixes  the error in settings page  ( it was beacuse the server didnt sent event_code for that event and we assigned the default event but there wasnt icon for it at the right place .  

it would be great if you would build this to check :)

**Comment by theBeginner86** [comment]
> @theBeginner86 this also fixes the error in settings page ( it was beacuse the server didnt sent event_code for that event and we assigned the default event but there wasnt icon for it at the right place .
> 
> it would be great if you would build this to check :)

Ohh, I see.
Looking....

**Review by theBeginner86 — COMMENTED** [review]
Looking good! UI doesn't break on receiving events.

Would you please confirm these discolored events. I believe, earlier these should to be of success type (that is green in color).


Screenshot 2023-08-19 at 4 43 13 PM


Screenshot 2023-08-19 at 4 41 13 PM

**Comment by aabidsofi19** [comment]
@theBeginner86  this the styling for default events ,  i can make info as default or we can add an event_code to this  on server

**Comment by theBeginner86** [comment]
> @theBeginner86 this is the default styling , we can make info as default or we can add an event_code to this on server

Hmm, if you can please make this specific notification as informational then it would be good.  👍

Lets merge after that considering this seems a high priority issue as other users might be facing the UI crash on receiving notification.

**Comment by aabidsofi19** [comment]
it is a bug somewhere in server , it already has info event type : 
image

**Comment by aabidsofi19** [comment]
@theBeginner86  i have made info as default for sse

File: ui/utils/hooks/useNotification.js [hunk 1]
```diff
@@ -9,33 +9,58 @@ import moment from "moment";
...
+   */
   const openEvent = (eventId) => {
     dispatch(toggleNotificationCenter())
     dispatch(openEventInNotificationCenter({
       eventId
     }))
   }
+
+  /**
+   * Notifies and stores the event.
```
Comment (leecalcote) [inline ui/utils/hooks/useNotification.js:39]: Good structure

**Review by theBeginner86 — APPROVED** [review]
Lgtm

**Comment by aabidsofi19** [comment]
@leecalcote  this needs merged

**Review by leecalcote — APPROVED** [review]

**Pull request merged**